src/utils/constants.ts

```diff
@@ -11,8 +11,15 @@ export const Ten = 10;
 export const ITEMS_PER_PAGE = 20;
 export const Thirty = 30;
 export const Hundred = 100;
-export const inputCharLimit = 254;
+
+export const DefaultInputCharLimit = 255;
```

Comment:
I appreciate you adding to this list. Might be something to import from [regtech-regex](https://github.com/cfpb/regtech-regex) someday in the future potentially, maybe after #811 is worked on.